tools/runqslower.py

```diff
@@ -207,6 +218,10 @@
 }
 """
 
+
+if args.pid and args.tid:
+    parser.error("specify only one of -p and -t")
```

Comment:
With exclusive group in the above, the above checking should be redundant, python should have warned earlier.

Reply:
I thought the same! I added this line because `tools/offcputime.py` had that too at line 102.
https://github.com/iovisor/bcc/blob/master/tools/offcputime.py#L102
Maybe that should be removed then too.
I will do so in mine.